src/OpenTelemetry.Exporter.InfluxDB/OpenTelemetry.Exporter.InfluxDB.csproj

```diff
@@ -1,16 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Description>An OpenTelemetry .NET exporter that exports to InfluxDB</Description>
-    <Authors>OpenTelemetry Authors</Authors>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>$(NetMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
+    <TargetFrameworks>net8.0;$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
```

Comment:
I propose to hardcode `net8.0` for the moment and reintroduce the property when `net6.0` will be completely removed. We can also introduce a new property now that will only contain `net8.0` if needed but not sure if it's relevant.